config.json

```diff
@@ -189,6 +189,16 @@
       ]
     },
     {
+      "uuid": "4a83a72c-db0a-45b6-b77c-1949cb24fbae",
+      "slug": "sublist",
+      "difficulty": 4,
+      "core": false,
+      "unlocked_by": null,
```

Comment:
We haven't decided yet about what exercises unlock what exercises, but in this case I think we could set `unlocked_by` property.

`sublist` is an exercises that makes you implement some functionality that is provided by some languages or libraries, and it's related to lists. Do you think could it be unlocked by `linked-list` exercise?

I think `linked-list` could unlock: circular-buffer, binary-search, custom-set and simple-linked-list